src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -194,9 +194,16 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
 
     // upload a file
     expectsUserFooBar
-    post(uri = s"/deposit/$uuid/file/path/to/text.txt", headers = Seq(basicAuthentication), body = randomContent(22)) {
+    val headers = Seq(
+      ("Content-Disposition", "text.txt"),
```

Comment:
the value of the Content-Disposition header is not the filename directly. 
`The first parameter [...] is always form-data. Additional parameters are case-insensitive and have arguments that use quoted-string syntax after the '=' sign. Multiple parameters are separated by a semi-colon (';').`
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

Reply:
From the api.yml:
`
    ContentDisposition:
      name: "Content-Disposition; filename"
      in: header
      description: the name to use for the new file in the deposit; required unless Content-Type is `application/zip`.
      required: true
      schema:
        type: "string"
`

in this spec, the equal-sign (=) seems to be missing